rfd/0155-scoped-webauthn-credentials.md

```diff
@@ -149,11 +152,37 @@ stored for the user has scope "headless", the verification will fail.
 
 #### Reuse
 
-After verifying a webauthn credential, the Auth server will check the `Uses`
-field of the stored challenge. If there are 0 (default) or 1 uses remaining,
-the challenge will be deleted and the client's webauthn credentials rendered
-useless. If there is more than 1 use remaining, the field will be decremented
-and updated in the backend.
+After verifying a webauthn credential, the Auth server will check the
+`AllowReuse` field of the stored challenge:
+
+- If `false`, the challenge will be deleted and the client's webauthn credentials
+rendered useless.
+- If `true`, the challenge will not be deleted.
+
+As mentioned in [Security](#security), reuse will only be permitted for the
```

Comment:
Just a nod to https://github.com/gravitational/teleport/pull/35185#discussion_r1412343453, which essentially suggests making reuse not allowed by default.